### PR TITLE
fix: Disable javascript when webview is not focused

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**
More info will be find on the [issue](https://github.com/MetaMask/mobile-planning/issues/1247#issuecomment-1810856723). 

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [ ] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by NicolasMassart — DISMISSED** [review]
Looks good to me.

File: app/components/Views/BrowserTab/index.js [hunk 1]
```diff
@@ -1429,6 +1432,16 @@ export const BrowserTab = (props) => {
     [reload],
   );
 
+  /**
```
Comment (Cal-L) [inline app/components/Views/BrowserTab/index.js:1435]: Change the comments to something like
```
According to Apple docs, it is not possible to update properties such as `javascriptEnabled` dynamically - https://developer.apple.com/documentation/webkit/wkwebviewconfiguration. By updating the key prop, we are forcing iOS WebView to reinitialize with the new `javascriptEnabled` value.
```

**Review by Cal-L — COMMENTED** [review]
left a comment

File: app/components/Views/BrowserTab/index.js (hunk 1 above)
Comment (tommasini) [reply app/components/Views/BrowserTab/index.js:1435]: Done! Much better! Thank you for finding this on the docs!

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**